ibm/mas_devops/common_vars/casebundles/v8-230407-amd64.yml

```diff
@@ -0,0 +1,60 @@
+---
+# Case bundle configuration for IBM Maximo Operator Catalog v230407
+# -----------------------------------------------------------------------------
+# In the future this won't be necessary as we'll be able to mirror from the
+# catalog itself, but not everything in the catalog supports this yet (including MAS)
+# so we need to use the CASE bundle mirror process still.
+
+catalog_digest: sha256:a2120ecbe06ff90d4c9f3be6cd32b8ddf6a44796e96da8af4bd325338d2b6980
+
+# Dependencies
+# -----------------------------------------------------------------------------
+common_svcs_version: 1.19.0   # Operator version 3.23.0 (https://github.com/IBM/cloud-pak/tree/master/repo/case/ibm-cp-common-services)
+db2u_version: 4.5.3           # Operator version 2.2.0 (https://github.com/IBM/cloud-pak/tree/master/repo/case/ibm-db2uoperator)
```

Comment:
We need to update this for v3.1 channel that we are going to start using now (for OCP 4.12 compatability)